build-system/test-configs/forbidden-terms.js

```diff
@@ -764,7 +764,6 @@ const forbiddenTermsGlobal = {
       'test/unit/test-intersection-observer-polyfill.js',
       'test/unit/test-json.js',
       'test/unit/test-layout-rect.js',
-      'test/unit/test-layout.js',
```

Comment:
Not related to this change: the regex for this term is incorrect since it misses `describe` methods after a newline:

```js
describe
  .configure()
  ...
```

This file **should** match. However, if I fix the pattern I'd have to allow a bunch of unrelated files so I'd rather fix that separately.